src/state-summary/state-card-configurator.js

```diff
@@ -31,7 +31,7 @@ class StateCardConfigurator extends LocalizeMixin(PolymerElement) {
 
       <!-- pre load the image so the dialog is rendered the proper size -->
       <template is="dom-if" if="[[stateObj.attributes.description_image]]">
-        <img hidden="" src="[[stateObj.attributes.description_image]]" />
+        <img hidden="" alt="" src="[[stateObj.attributes.description_image]]" />
```

Comment:
I have no idea where to find this in the frontend.  I presume an empty string was fine since it has the `hidden` attribute anyway.  Please confirm.

Reply:
So here it's only to preload the image, this will not actually be shown. 

The `configurator` is how we used to configure things before config flow 🙈  very old. This is fine here.